tests/browser/test_close.py

```diff
@@ -18,6 +18,7 @@
 
 
 @pytest.mark.asyncio
+@pytest.mark.skip(reason="Flaky. TODO: Fix this test")
```

Comment:
Please open an issue for tracking.

Reply:
done

Reply:
Based on the stacktrace of https://github.com/GoogleChromeLabs/chromium-bidi/issues/1964, it can be unrelated to the specific test. Should be investigated in the issue.